test/verify/check-packagekit

```diff
@@ -371,7 +371,7 @@ class TestUpdates(NoSubManCase):
         assertHistory("table.updates-history ul", ["secdeclare", "secparse", "sevcritical"])
 
         # stop PackageKit (e. g. idle timeout) to make sure the page survives that
-        m.execute("systemctl kill --signal=KILL packagekit.service")
+        m.execute("systemctl stop packagekit && systemctl reset-failed packagekit")
```

Comment:
This looks a little curious -- the `reset-failed` will *not* run if  `stop` fails, due to the shell's rather strange "false && true" behaviour (the compound command does not fail under `set -e`, but due to short-circuiting the `true` does not run)

Reply:
Good point, fixed. The restart is actually needed at least for the new f-testing.